### DISCUSSION TITLE
ScriptReference/MonoBehaviour.StartCoroutine

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#531] [Notes]
# ScriptReference/MonoBehaviour.StartCoroutine

https://docs.unity3d.com/ScriptReference/MonoBehaviour.StartCoroutine.html

**Comment by joonjoonjoon** [comment]
tiny thing that's buried in the comments of the examples:

**yield return MyIenumerator();**
is the same as
**yield return StartCoroutine(MyIenumerator());**
